applications/cryptex.md

```diff
@@ -0,0 +1,297 @@
+# Cryptex: EtF Network with Aura 
+
+- **Team Name:** Ideal Labs
+- **Payment Address:** 0x2CDA3C7D6e21Cc4f43C170c0dFf2e9F3B3B5E889 (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+This proposal presents a blockchain that uses a modified Aura that seals blocks using identity based signatures (BLS). We then implement an encryption-to-the-future (EtF) scheme, where messages can be encrypted for arbitrary slots and epochs in the future. Our proposal consists of a runtime, which modifies Aura and introduces a new pallet to enable the identity based cryptosystem (IBC), a light client, and an SDK which handles synchronization with the blockchain, slot scheduling, and offchain encryption and decryption functionality. In essence, this presents a simplified flavor of 'timelock encryption', though sacrifices full-decentralization for ease of implementation and feasibility (for the time being). 
+
+Being the first EtF network in the ecosystem, Cryptex introduces several new cryptographic primitives which would be useful to others as well. This proposal lays the foundation for a more robust system later on, using a proof of stake consensus (Sassafras) and more sophisticated cryptographic primitives for EtF, such as [McFly](http://fc23.ifca.ai/preproceedings/189.pdf) or based on [commitment witness encryption](https://eprint.iacr.org/2021/1423.pdf). An EtF network can enable randomness beacons, sealed-bid auctions, non-interactive secret sharing, and many other use cases.
+
+We want to build more extensive and secure decentralized data tools, for which this proposal lays the foundation. We believe that the internet is a better place when it's more fair for all.
+
+### Project Details
+
+The major pieces:
+1. [IBS Block Seal](#ibe-block-seal-in-aura)
+2. [Encryption-to-Future](#encryption-to-future-slots)
+
+For a more in-depth (informal) look at the mathematics behind this, look at [etf_aura.pdf](https://github.com/ideal-lab5/Grants-Program/blob/ibe/applications/etf_aura.pdf)
+
+#### What this is not
+- this does not use a proof of stake consensus. For the scope of the proposal, we are assuming a static, well defined validator set using PoA consensus based on Aura. 
+- the proposal does not highlight any specific privacy preserving tools nor does it use threshold signatures or any specific MPC protocols 
+- most of that is scoped for the future though
```

Comment:
This is more a substrate concern of parity not having made proof-of-stake easily available yet.  There is nothing really wrong with using proof-of-stake here, but like VRF protocols you need to know who'll be the nodes before chosing the session id, or at leaast before the users use it.  Nothing wrong here, just observing.

Reply:
I left the comment in for now, just so we can lay down expectations/scope